sdk/include/psdk/textserv.h

```diff
@@ -44,10 +55,10 @@ DECLARE_INTERFACE_(ITextServices,IUnknown)
 
     /*** ITextServices methods ***/
 
-    STDMETHOD(TxSendMessage)( THIS_
+    THISCALLMETHOD_(HRESULT,TxSendMessage)( THIS_
```

Comment:
So of course, the reason why here they try to use "THISCALLMETHOD", is because the interface declaration, in the official MS PSDK headers, is:
```c
class ITextServices : public IUnknown
{
...
}
```
i.e. this header supposes it's being used within C++ code only.